tests/jenkins/TestCopyContainer.groovy

```diff
@@ -0,0 +1,92 @@
+import jenkins.tests.BuildPipelineTest
+import org.junit.*
+
+class TestCopyContainer extends BuildPipelineTest {
+
+    String sourceImage = 'alpine:3.15.4'
+    String destinationImage = 'alpine:3.15.4'
+
+    @Before
+    void setUp() {
+        binding.setVariable('DOCKER_USERNAME', 'dummy_docker_username')
+        binding.setVariable('DOCKER_PASSWORD', 'dummy_docker_password')
+        binding.setVariable('ARTIFACT_PROMOTION_ROLE_NAME', 'sample-agent-AssumeRole')
+        binding.setVariable('AWS_ACCOUNT_ARTIFACT', '1234567890')
+         helper.registerAllowedMethod('withAWS', [Map, Closure], null)
+        super.setUp()
+
+    }
+
+    @Test
+    public void testCopyContainerDockerStagingToDockerProd() {
+
+        binding.setVariable('SOURCE_IMAGE_REGISTRY', 'opensearchstaging')
+        binding.setVariable('SOURCE_IMAGE', sourceImage)
+        binding.setVariable('DESTINATION_IMAGE_REGISTRY', 'opensearchproject')
+        binding.setVariable('DESTINATION_IMAGE', destinationImage)
+
+        super.testPipeline("jenkins/docker/docker-copy.jenkinsfile",
+                "tests/jenkins/jenkinsjob-regression-files/docker-ecr/docker-copy-testCopyContainerDockerStagingToDockerProd.jenkinsfile")
```

Comment:
Since we removed the `docker-ecr` folder in jenkins folder, we should just rename this as `docker`, not `docker-ecr`.

Reply:
fixed in my latest commit @peterzhuamazon